test/src/vfs_helpers.h

```diff
@@ -182,6 +195,13 @@ class SupportedFsS3 : public SupportedFs {
    */
   virtual std::string temp_dir();
 
+  /**
+   * Get the name of the filesystem's bucket
+   *
+   * @return string bucket name
+   */
+  virtual std::string bucket();
```

Comment:
it looks like this is unused (i assume it was going to be used for unit-capi-vfs/unit-vfs.cc). let's remove this for now since we're not using it